composeApp/src/androidMain/kotlin/com/wisemen/chargehub/MainActivity.kt

```diff
@@ -2,23 +2,30 @@ package com.wisemen.chargehub
 
 import App
 import android.os.Bundle
+import android.util.Log
 import androidx.activity.ComponentActivity
 import androidx.activity.compose.setContent
 import androidx.compose.runtime.Composable
 import androidx.compose.ui.tooling.preview.Preview
+import db.chargehub.User
+import db.repository.user.RemoteUserRepository
+import org.koin.core.component.KoinComponent
+import org.koin.core.component.inject
+
+class MainActivity : ComponentActivity(), KoinComponent {
 
-class MainActivity : ComponentActivity() {
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
+        val userRepo: RemoteUserRepository by inject()
 
         setContent {
-            App()
+            App(userRepo.getAll().map { it.name }.firstOrNull() ?: "Nothing found in db")
         }
     }
 }
 
 @Preview
 @Composable
 fun AppAndroidPreview() {
-    App()
+    //App()
```

Comment:
Why did you comment out this in the preview? Couldn't you just add a string to make the preview work?